### ISSUE TITLE
NTR: Palmar creases, hockey-stick (Extremities, joints & nails)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7972]
New term request Palmar creases, hockey-stick (elthat_120816231721):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000112
- label: Hockey-stick palmar crease
- parents: [Abnormality of the palmar creases (HP:0010490)](https://hpo.jax.org/app/browse/term/HP:0010490)
- pmid: PMID:29094891
- def: An abnormality of the upper palmar crease whereby the crease widens and ends between the second and third fingers and is said to resemble a hockey stick.
- comment: See figure 5 of PMID:29094891.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7972
- synonyms: Hockey-stick crease(NONE)
